pkg/machine/machine_windows_test.go

```diff
@@ -0,0 +1,137 @@
+//go:build windows
+
+package machine
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+)
+
+// CreateNewItemWithPowerShell creates a new item using PowerShell.
+// It's an helper to easily create junctions on Windows (as well as other file types).
+// It constructs a PowerShell command to create a new item at the specified path with the given item type.
+// If a target is provided, it includes it in the command.
+//
+// Parameters:
+//   - path: The path where the new item will be created.
+//   - itemType: The type of the item to be created (e.g., "File", "SymbolicLink", "Junction").
+//   - target: The target for the new item, if applicable.
+func CreateNewItemWithPowerShell(path string, itemType string, target string) {
+	var pwshCmd string
+	pwshCmd = "New-Item -Path " + path + " -ItemType " + itemType
+	if target != "" {
+		pwshCmd += " -Target " + target
+	}
+	cmd := exec.Command("pwsh", "-Command", pwshCmd)
+	if err := cmd.Run(); err != nil {
+		log.Fatal(err)
```

Comment:
`log.Fatal()` is not how to deal with errors in tests
The proper go std thing is to use `t.Fatal()` however I suggest you switch to using the assert/require package instead.
```
err := cmd.Run()
require.NoError(t, err)
```

same in all other palces here

Reply:
fixed thanks